app/services/food_service.rb

```diff
@@ -0,0 +1,8 @@
+class FoodService < BaseService
+  def self.get_food_data
+    response = conn("https://api.nal.usda.gov").get("/fdc/v1/foods/search?api_key=food_api_key&query=")
```

Comment:
where is the food_api_key stored? This looks like it's hard-coded rather than pulling from an ENV variable like figaro does. Try to use figaro instead and see if you can get it working.